.github/workflows/test-move-prover.yml

```diff
@@ -26,7 +26,10 @@ jobs:
     - name: Checkout libra
       uses: actions/checkout@v2
       with:
-        path: 'libra'
+        path: "libra"
+        fetch-depth: 50 # this is to make sure we obtain the target base commit
```

Comment:
you don't need this (I was using this to checkout the BASE but you don't do that)